README.md

```diff
@@ -22,69 +25,186 @@ Upsilon is a fork of Omega, an user-made OS that runs on the Numworks calculator
+```
+and it will install all dependencies.
+
+<br>
+
+And there you can go to step 2!
+
+<br>
+
+
+
+</details>
+
+<details>
+<summary><b>1.3 Windows</b></summary>
+
+<br>
+
+[Msys2](https://www.msys2.org/) environment is recommended to get most of the required tools on Windows easily. It's where you'll paste all the commands of this tutorial. Once it's installed, paste these commands into the Msys2 terminal.
+
+```bash
+pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-freetype mingw-w64-x86_64-pkg-config mingw-w64-x86_64-libusb git make python
+echo "export PATH=/mingw64/bin:$PATH" >> .bashrc
+```
+
+Next, you'll need to install the [GCC toolchain for ARM](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads). When prompted for an install location, choose `C:\msys64\home\User\gcc-arm\`. You'll then need to add this folder to your $PATH. Just enter:
+
+```bash
+echo "export PATH=$PATH:$HOME/gcc-arm/bin" >> .bashrc
+```
+Just restart and you can go to step 2!
+</details>
+
+<br>
+
+### 2. Set up repo
+
+
+Clone repo and use 'upsilon-dev' branch by pasting these two commands:
 
 ```bash
 git clone --recursive https://github.com/Lauryy06/Upsilon.git
 cd Upsilon
-git checkout omega-master
+git checkout upsilon-dev
+```
+<br>
+
+
+### 3.Choose the target
+
+
+<details>
+  <summary><b>Model n0100</b></summary>
+
+(note: you can change the `EPSILON_I18N=en` flag to `fr`, `nl`, `pt`, `it`, `de`, `es` or `hu`).
+
+```bash
 make MODEL=n0100 clean
 make MODEL=n0100 EPSILON_I18N=en OMEGA_USERNAME="{Your name, max 15 characters}" -j4
+```
+
+Now, run either:
+
+```bash
 make MODEL=n0100 epsilon_flash
 ```
+to directly flash the calculator after pressing simultaneously `reset` and `6` buttons and pluging in.
 
-Important: Don't forget the `--recursive` tag, because Omega relies on submodules.
-Also, you can change the number of processes that run in parallel during the build by changing the value of the `-j` flag.
+<br>
+
+or:
+
+```bash
+make MODEL=n0100 OMEGA_USERNAME="" binpack -j4
+```
+to make binpack wich you can flash to the caculator from [Ti-planet's webDFU](https://ti-planet.github.io/webdfu_numworks/n0100/). Binpacks are a great way to share a custom build of Upsilon to friends.
   
 </details>
 
 <details>
   <summary><b>Model n0110</b></summary>
 
+
 ```bash
-git clone --recursive https://github.com/Lauryy06/Upsilon.git
-cd Upsilon
-git checkout omega-master
 make clean
 make OMEGA_USERNAME="{Your name, max 15 characters}" -j4
+```
+
+Now, run either:
+
+```bash
 make epsilon_flash
 ```
+to directly flash the calculator after pressing simultaneously `reset` and `6` buttons and pluging in.
 
-Important: Don't forget the `--recursive` tag, because Omega relies on submodules.
-Also, you can change the number of processes that run in parallel during the build by changing the value of the `-j` flag.
-  
-</details>
+<br>
 
-<details>
-  <summary><b>Bin files</b></summary>
```

Comment:
You don't put this part in the new version. It's your choice or a mistake ?

Reply:
Achoice because it sounds more simple and logic to get binpacks directly in device section.